src/App.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { useEffect, useState } from "react";
```

Comment:
동일 패키지에서 import하셨다면 같이 써주시면 됩니다.
```suggestion
import React, { useEffect, useState } from "react";
```